### PR TITLE
[WIP]Try to fix conda env

### PR DESCRIPTION
**Pull request by yinweisu**
*Issue #, if available:*

*Description of changes:*
* Get rid of removal and always create a new environment.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-1638-7 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1638/7/index.html

File: Jenkinsfile [hunk 1]
```diff
@@ -196,8 +191,7 @@ stage("Unit Test") {
           VISIBLE_GPU=env.EXECUTOR_NUMBER.toInteger() % 8
           sh """#!/bin/bash
           set -ex
-          conda remove --name autogluon-vision-py3 --all -y
-          conda env update -n autogluon-vision-py3 -f docs/build_gpu.yml
+          conda env update -n autogluon-vision-py3-v3 -f docs/build_gpu.yml --prune
```
Comment (Innixma) [inline Jenkinsfile:194]: Incorrect: This should be `autogluon-vision-py3`

**Comment by szha** [comment]
Job PR-1638-20 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1638/20/index.html

**Comment by gradientsky** [comment]
Conda `create` doesn't make a fresh environment. We should keep `remove` to have a fresh environment for reproducible builds:
```shell
$ conda create -n tmp -y
$ conda activate tmp
$ pip freeze | grep numpy
numpy @ file:///opt/concourse/worker/volumes/live/a1b5ea96-b05c-40ba-5dfb-4542ece1c186/volume/numpy_and_numpy_base_1603491228242/work

$ pip install -U numpy
Requirement already satisfied: numpy in ./anaconda3/lib/python3.7/site-packages (1.19.2)
Collecting numpy
  Using cached numpy-1.21.5-cp37-cp37m-macosx_10_9_x86_64.whl (16.9 MB)
Installing collected packages: numpy
  Attempting uninstall: numpy
    Found existing installation: numpy 1.19.2
    Uninstalling numpy-1.19.2:
      Successfully uninstalled numpy-1.19.2
Successfully installed numpy-1.21.5

$ pip freeze | grep numpy
numpy==1.21.5

$ conda deactivate
$ conda create -n tmp -y
$ conda activate tmp
$ pip freeze | grep numpy
numpy==1.21.5
```

**Review by gradientsky — APPROVED** [review]

**Pull request merged**